code/modules/vending/_vending.dm

```diff
@@ -207,6 +207,12 @@
 	/// used for narcing on underages
 	var/obj/item/radio/sec_radio
 
+/// Contains structures and items that vendors shouldn't crush when we land on them.
+GLOBAL_LIST_INIT(vendor_uncrushable_items, list(
```

Comment:
Like Melbert and I discussed, make this a static/list defined in the scope of the can_be_vendorcrushed proc

Reply:
> Like Melbert and I discussed, make this a static list stored in the scope of the can_be_vendorcrushed proc. Earlier reviews contained a code snippet showing you how to do it

Oh, so I SHOULD make it static. sorry, misunderstanding.

Reply:
You need to move it to the scope of (or inside) the vendorcrush check proc, because otherwise it can't find the list